common/src/IO/EntParser.cpp

```diff
@@ -622,6 +622,36 @@ std::optional<Color> EntParser::parseColor(
   return Color::parse(parseString(element, attributeName, status));
 }
 
+std::optional<bool> EntParser::parseBoolean(
+  const tinyxml2::XMLElement& element,
+  const std::string& attributeName,
+  ParserStatus& /* status */)
+{
+  std::string strValue = element.Attribute(attributeName.c_str());
```

Comment:
I will, since it is not my project, but if I may express my opinion, I think using `auto` all the time just ends up making the code harder to understand. I still use it, though, when the type can easily be guessed or is not important for the following operations, i.e. in range-for or with casts.
This is just my opinion, of course, and again: not my project, so I'll conform to codestyle.